src/utils.js

```diff
@@ -0,0 +1,90 @@
+var _ = require('lodash');
+var Q = require('q');
+var vumigo = require('vumigo_v02');
+var HttpApi = vumigo.http.api.HttpApi;
+
+function format_address(address) {
+  if (isNoU(address) || address.length === 0) {
+    return '';
+  }
+
+  var len = address.length;
+  var last_4_digits = address.slice(-4, len);
+  return (len > 4 ? Array(len - 3).join('*') : '') + last_4_digits;
+}
+
+function send_email(email_config, usr_addr, im) {
+  return Q.promise(function (resolve, reject) {
```

Comment:
I don't think you need the `Q.promise` wrapper here because `api.post(...)` already returns a promise.